blocks/cards/hero-profile.css

```diff
@@ -0,0 +1,76 @@
+/* Profile View Styles */
+
```

Comment:
dont create separate css file, it will lead to additional CSS call

Reply:
Done. removed the file